tests/pytests/test_multithread.py

```diff
@@ -469,48 +469,55 @@ def expected(conns):
     env.expect(debug_cmd(), 'SHARD_CONNECTION_STATES').equal(expected(1))
 
 def test_change_workers_number():
+
+    def check_threads(expected_num_threads_alive, expected_n_threads):
+        env.assertEqual(getWorkersThpoolStats(env)['numThreadsAlive'], expected_num_threads_alive)
+        env.assertEqual(getWorkersThpoolNumThreads(env), expected_n_threads)
     # On start up the threadpool is not initialized. We can change the value of requested threads
     # without actually creating the threads.
     env = initEnv(moduleArgs='WORKER_THREADS 1 MT_MODE MT_MODE_FULL')
-    env.assertEqual(getWorkersThpoolStats(env)['numThreadsAlive'], 0)
+    check_threads(0, 1)
```

Comment:
consider using keyword syntax:
```suggestion
    check_threads(expected_num_threads_alive=0, expected_n_threads=1)
```